.github/workflows/android_unit_test.yml

```diff
@@ -0,0 +1,32 @@
+name: Android Unit Tests
+
+on:
+  pull_request:
+    branches:
+      - develop
+  push:
+    branches:
+      - develop
+
+
+jobs:
+  Unit-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with: { java-version: 11 }
+
+      - name: Setup google-json from secrets
+        env:
+          FIREBASE_SECRET: ${{ secrets.GOOGLE_SERVICE_JSON_SECRET }}
+        run: echo $FIREBASE_SECRET > app/google-services.json
```

Comment:
is it hex of the secret or just the raw file?

Reply:
raw file